src/main/antora/modules/ROOT/pages/jpa/misc-merging-persistence-units.adoc

```diff
@@ -37,5 +37,5 @@ A plain JPA setup requires all annotation-mapped entity classes to be listed in
 ----
 ====
 
-NOTE: As of Spring 3.1, a package to scan can be configured on the `LocalContainerEntityManagerFactoryBean` directly to enable classpath scanning for entity classes. See the link:{springJavadocUrl}org/springframework/orm/jpa/LocalContainerEntityManagerFactoryBean.html#setPackagesToScan(java.lang.String...)$$[JavaDoc] for details.
+NOTE: As of Spring 3.1, a package to scan can be configured on the `LocalContainerEntityManagerFactoryBean` directly to enable classpath scanning for entity classes. See the link:{springJavadocUrl}/org/springframework/orm/jpa/LocalContainerEntityManagerFactoryBean.html#setPackagesToScan(java.lang.String...)$$[JavaDoc] for details.
```

Comment:
![스크린샷 2024-05-28 오후 10 25 00](https://github.com/spring-projects/spring-data-jpa/assets/37570657/60cf0515-5890-4ed3-ba6f-13d19a726424)

The link is broken because it should be javadoc-api/org instead of javadoc-apiorg.